src/SecureStream.php

```diff
@@ -4,12 +4,11 @@
 
 use Evenement\EventEmitterTrait;
 use React\EventLoop\LoopInterface;
-use React\Stream\DuplexStreamInterface;
 use React\Stream\WritableStreamInterface;
 use React\Stream\Stream;
 use React\Stream\Util;
 
-class SecureStream extends Stream implements DuplexStreamInterface
+class SecureStream extends Stream
```

Comment:
This actually amends #26 which in turn amends #14.

The documentation always states this returns a `Stream` instance, never an instance implementing `DuplexStreamInterface`.

FWIW: This repo currently targets `react/stream:0.4.*`, while this interface has only been added with `v0.4.2`, so even if we decide not to support legacy versions, this is still a bug that needs fixing. If you happen to have `v0.4.2+`, then `Stream` implements the `DuplexStreamInterface`.

As such, I do not consider this a BC break.

Even despite this, this PR should probably be part of the `v0.5.0` milestone, which already includes a (minor) BC break anyway.